modules/server_new/src/main/scala/observe/server/keywords/StandardHeader.scala

```diff
@@ -73,7 +73,7 @@ class StandardHeader[F[_]: Sync: Logger](
       buildString(obsObject, KeywordName.OBJECT),
       buildString(obsReader.obsType, KeywordName.OBSTYPE),
       buildString(obsReader.obsClass, KeywordName.OBSCLASS),
-      buildString(dataset.map(_.observation.program.label.label).getOrElse("").pure[F],
+      buildString(dataset.map(_.observation.label.programReference.label).getOrElse("").pure[F],
```

Comment:
Does this return the same value? Maybe we can leave it like this.

Reply:
it should be the same